code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -292,6 +304,7 @@
 /datum/tarot/the_high_priestess
 	name = "II - The High Priestess"
 	desc = "Mother is watching you."
+	extended_desc = "alerts Bubblegum to the user, allowing them to rend them, dealing heavy damage, and imobilizing them."
```

Comment:
```suggestion
	extended_desc = "alerts Bubblegum to the user, allowing them to rend them, dealing heavy damage, and imobilizing them."
```
who is them? Pronouns here are not being used in a particularly helpful way

Reply:
reworded it slightly